pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+addopts = --doctest-modules
```

Comment:
I enabled doctest for testing purpose. You would only use doc test for simple functions (mostly logic based) and therefore no doctest is written for your data code